Makefile

```diff
@@ -72,19 +72,23 @@ help:
 	@echo "- format             Format the python source code"
 	@echo "- lint               Lint the python source code"
 	@echo "- format-lint        Format and lint the python source code"
+	@echo "- lint-spec          Lint the openapi spec"
 	@echo "- test               Run the tests"
 	@echo -e " \033[1mLOCAL SERVER TARGETS\033[0m "
 	@echo "- serve              Run the project using the flask debug server. Port can be set by Env variable HTTP_PORT (default: 5000)"
 	@echo "- gunicornserve      Run the project using the gunicorn WSGI server. Port can be set by Env variable DEBUG_HTTP_PORT (default: 5000)"
+	@echo "- serve-spec-redoc   Serve the spec using Redoc on localhost:8080"
+	@echo "- serve-spec-swagger Serve the spec using Swagger on localhost:8080/swagger"
 	@echo -e " \033[1mDocker TARGETS\033[0m "
 	@echo "- dockerlogin        Login to the AWS ECR registery for pulling/pushing docker images"
 	@echo "- dockerbuild        Build the project localy (with tag := $(DOCKER_IMG_LOCAL_TAG)) using the gunicorn WSGI server inside a container"
 	@echo "- dockerpush         Build and push the project localy (with tag := $(DOCKER_IMG_LOCAL_TAG))"
 	@echo "- dockerrun          Run the project using the gunicorn WSGI server inside a container (exposed port: 5000)"
-	@echo "- shutdown           Stop the aforementioned container"
 	@echo -e " \033[1mCLEANING TARGETS\033[0m "
 	@echo "- clean              Clean genereated files"
 	@echo "- clean_venv         Clean python venv"
+	@echo "- clean_logs			Clean logs"
+	@echo "- clean_vol_minio	Clean $(VOLUMES_MINIO)"
```

Comment:
Be aware of space/tabs for code alignment, uses here space instead of tabs in the string.